api/test/integration/test_agent_GET_endpoints.tavern.yaml

```diff
@@ -1898,6 +1898,42 @@ stages:
             rootcheck:
               sleep: !anyint
 
+  - name: Request-Wazuh-db-Internal
```

Comment:
This name format doesn't match the standard. I don't know why we're maintaining it.

Reply:
I have updated the format of all names